### ISSUE TITLE
[Q] Is there is away set general rules for wandb.define_metric

### ISSUE THREAD

**Issue by MoH-assan** [wandb/wandb#6875]
Hi, 
Instead of defining every individual metric, is there is away to set more general rule, something like

summary="min" if "loss" in metric_name

**Comment by rsanandres-wandb** [comment]
Hello! There isn't a way to set a general rule with define rule with `define_metric` since we would need all the data surrounding each metric in order to full graph it in the UI. If you want to look into the details of `define_metric`, [here](https://github.com/wandb/wandb/blob/722f9737ce1a77b8970fef275047e8a0f4a1a68e/wandb/sdk/wandb_run.py#L2556) is the exact line of code we run in order to make a new metric.

**Comment by rsanandres-wandb** [comment]
Correct, the `define_metric` function does need to have the metric defined already. I can certainly add in the second point to the feature request.

**Comment by sydholl** [comment]
WandB Internal User commented: 
MoH-assan commented: 
Thanks, but this would be a nice feature. 
Or allow the rule to be defined when doing wandb.log the first time the metric is logged or something like that. 

If I am not wrong the current implementation requires knowing the names of the metrics to be logged and individually defining the rule for them before logging them, right?
